hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -99,6 +99,26 @@
   cov_report_txt:       "{cov_report_dir}/dashboard.txt"
   cov_report_page:      "dashboard.html"
 
+  // UNR related.
+  // All code coverage, assert isn't supported
+  cov_unr_metrics: "line+cond+fsm+tgl+branch"
+  cov_unr_dir:     "{scratch_base_path}/cov_unr"
+
+  cov_unr_common_build_opts: "-ntb_opts uvm-1.2 -sverilog -full64 -timescale=1ns/1ps"
+  // Use UUM (Unified usage model) 3 steps flow, so that we can specify the dut and use an unified
+  // unr.cfg. One step flow requires to specify the dut top in the cfg file
+  cov_unr_build_cmd: ["{job_prefix} vlogan {cov_unr_common_build_opts};",
+                      "{job_prefix} vlogan {cov_unr_common_build_opts} {build_defines} -f {sv_flist};",
```

Comment:
Perhaps try this? 

```suggestion
  cov_unr_build_cmd: ["{job_prefix} vlogan {cov_unr_common_build_opts} && ",
                      "{job_prefix} vlogan {cov_unr_common_build_opts} {build_defines} -f {sv_flist} && ",
```

With semi-colon, I wonder if errors from the first command causes the second (and third) command to be skipped (I think not).

Reply:
yes, this is better. Thanks